chrome_extension/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "openadapt",
+  "description": "Native messaging example add-on",
```

Comment:
```suggestion
  "description": "Uses sockets to expose DOM events to OpenAdapt",
```

Reply:
I believe we are using sockets instead of native messaging, please correct me if I'm wrong 🙏

Reply:
Yes, we are using SOCKET PROTOCOL instead of NATIVE MESSAGING.